java/ql/src/experimental/Security/CWE/CWE-470/LoadClassNoSignatureCheck.ql

```diff
@@ -70,16 +70,16 @@ predicate isSignaturesChecked(MethodAccess maCreatePackageContext)
 }
  
 from
-  MethodCall maCreatePackageContext, LocalVariableDeclExpr lvdePackageContext,
-  Expr sinkPackageContext, MethodCall maGetMethod, MethodCall maInvoke
+  MethodAccess maCreatePackageContext, LocalVariableDeclExpr lvdePackageContext,
+  DataFlow::Node sinkPackageContext, MethodAccess maGetMethod, MethodAccess maInvoke
 where
   maCreatePackageContext
       .getMethod()
       .hasQualifiedName("android.content", ["ContextWrapper", "Context"], "createPackageContext") and
   not isSignaturesChecked(maCreatePackageContext) and
   lvdePackageContext.getEnclosingStmt() = maCreatePackageContext.getEnclosingStmt() and
-  TaintTracking::localExprTaint(lvdePackageContext.getAnAccess(), sinkPackageContext) and
-  getClassLoaderReachableMethodCall(sinkPackageContext) = maGetMethod and
-  getGetMethodMethodCall(maGetMethod) = maInvoke
+  TaintTracking::localTaint(DataFlow::exprNode(lvdePackageContext.getAnAccess()), sinkPackageContext) and
```

Comment:
Changed
`TaintTracking::localExprTaint(lvdePackageContext.getAnAccess(), sinkPackageContext) and`
to
`TaintTracking::localTaint(DataFlow::exprNode(lvdePackageContext.getAnAccess()), sinkPackageContext) and`
to resolve the following error
`Error: Failed to run query: ERROR: DataFlowNodes::Public::Node is not compatible with Expr::Expr`